awx/ui/client/legacy/styles/lists.less

```diff
@@ -587,6 +587,11 @@ table, tbody {
 }
 
 .List-staticColumnLayout--groups {
+    display: grid;
+    grid-template-columns: 5px 25px auto;
+}
+
+.List-staticColumnLayout--inventoryGroups {
     display: grid;
```

Comment:
the `display: grid` line is redundant, you can remove it.

Reply:
@jlmitch5 good catch.  I have removed it.